espflash/src/error.rs

```diff
@@ -125,7 +125,7 @@ pub enum Error {
 
     #[error("This command requires using the RAM stub")]
     #[diagnostic(
-        code(espflash::stub_required_to_erase_flash),
```

Comment:
Renamed the code, as it also used for other non-`erase-flash` commands, like `read-flash`